src/functions.php

```diff
@@ -227,3 +227,32 @@ function percentile($arr, $percentile = 0.95)
     sort($arr);
     return $arr[max(round($percentile * count($arr) - 1.0 - $percentile), 0)];
 }
+
+/**
+ * Download the given URI and decode it as JSON.
+ * @param string $uri
+ *
+ * @return mixed
+ */
+function getURIContentAsJson($uri){
+    // get the json file
+    $httpsProxy = getenv('https_proxy');
+    if (!empty($httpsProxy)) {
+        return json_decode(@file_get_contents($uri, false, [
+            'http' => [
+                'proxy'           => str_replace(
+                    'https://',
+                    'tcp://',
+                    str_replace(
+                        'http://',
+                        'tcp://',
+                        $httpsProxy
+                    )
+                ),
+                'request_fulluri' => true,
+            ],
+        ]));
+    } else {
```

Comment:
`return` is unskippable in the `if` statement, so there's no need for `else`.

Also, as the `if` condition is negative, I would more inverse the structure and look at it in a positive way. This would improve the readability also as if the env variable is not set, simply return without creating a context, otherwise, (you don't need the `else`) create the context and use it.

Reply:
I think it is solved with the latest push, but let me know if you want me to work it out a bit more.